microovn/ovn/bootstrap.go

```diff
@@ -131,7 +132,7 @@ func Bootstrap(s *state.State, initConfig map[string]string) error {
 		return err
 	}
 
-	_, err = VSCtl(
+	_, err = cli.VSCtl(
```

Comment:
The imported name is a bit ambiguous, should we perhaps use the golang alias functionality and import the package with a prefix so that it could read `ovn_cmd.VSCtl()`